NServiceBus and CloudEvents landing page (#6834)
Particular/docs.particular.net: shape-the-future/index.md

---

> title: Shape the future with us

Reviewer: ```suggestion
title: Help us shape the future
```

summary: Provide feedback for the future of the Particular Service Platform
reviewed: 2024-09-16
---

> We are interested in your feedback regarding some capabilities that we are considering integrating into the Particular Service Platform.

Reviewer 2: ```suggestion
We are interested in your feedback regarding the future of the Particular Service Platform.
```

> We are interested in your feedback regarding some technologies that we are currently looking at integrating into the Particular Service Platform.

Reviewer: ```suggestion
We are interested in your feedback regarding some capabilities that we are considering integrating into the Particular Service Platform.
```

> This section contains some ideas that we have but if you have some other ideas that are not listed here please [tell us more](https://github.com/Particular/NServiceBus/issues/new?assignees=&labels=Feature&projects=&template=feature_request.yml).

Reviewer: ```suggestion
This section contains some capabilities that we're currently considering, but if you have some other ideas that are not listed here, please [tell us more](https://github.com/Particular/NServiceBus/issues/new?assignees=&labels=Feature&projects=&template=feature_request.yml).
```
